src/odemis/gui/test/comp_canvas_test.py

```diff
@@ -627,12 +627,10 @@ def test_pyramidal_zoom(self):
         numpy.testing.assert_almost_equal(init_pos, self.view.view_pos.value)
         # remove green picture
         result_im = get_image_from_buffer(self.canvas)
-        #result_im.SaveFile('/home/gstiebler/Projetos/Delmic/tmp3.bmp', wx.BITMAP_TYPE_BMP)
```

Comment:
Don't commit changes to comp_canvas_test, as they are not related to this branch. (Moreover, this will conflict with the latest master).